lib/libc/Kconfig

```diff
@@ -239,7 +239,20 @@ config LIBC_TZDIR
 	string "zoneinfo directory path"
 	default "/etc/zoneinfo"
 
-endif
+config LIBC_DOWNLOAD_ZONEINFO
+	bool "Download TZ database"
+	default y
+	---help---
+		Download and build the TZ/Olson database.
+
+config LIBC_ZONEINFO_ROMFS
+	bool "Build ROMFS filesystem"
+	default n
```

Comment:
I think we should consider the dependency of LIBC_DOWNLOAD_ZONEINFO and this config.
Because if someone wants to download the zoneinfo, it should be saved somewhere.

Reply:
yes. thanks. i will think more and modify.